docs/source/api/user-guide/context/index.rst

```diff
@@ -20,6 +20,7 @@ it can raise an exception and prevent you from running more queries than your bu
     .. code:: python
 
       >>> import opendp.prelude as dp
+      >>> dp.enable_features("contrib")
```

Comment:
@raprasad : If our testing is working correctly, this code sample ran without `dp.enable_features("contrib")` -- do we want to include it for the sake of consistency? Or do you think this example just should not work without `contrib` enabled? 

If the latter case, either the test framework itself is broken, or else the context code should add a check for `contrib`.